www/src/utils/array.ts

```diff
@@ -51,7 +51,7 @@ export function* chunk(array: any[], chunkSize) {
 }
 
 export function trimSuffix(str, suff) {
-  if (str.endsWith(suff)) {
+  if (str && str.endsWith(suff)) {
```

Comment:
nit: `?` can be also used here.

Reply:
Actually, it is not used anymore and can be deleted.